### ISSUE TITLE
Every yaml file is detected as Helm inside a project containing Chart.yaml

### ISSUE THREAD

**Issue by eshepelyuk** [towolf/vim-helm#23]
Hello @towolf @luisdavim 

After the latest update https://github.com/towolf/vim-helm/commit/69a51170f9e0f7c21242c85e2feac6aa17adeb22

Every `*.yaml` file within the folder containing `Chart.yaml` is detected as `helm`.
Even `Chart.yaml` itself is detected as `helm`

E.g. in such structure, all three `yaml` files are detected as `helm`, and that's wrong. 

Only ones located in `templates` folder should be considered as `helm`.

```
├── Chart.yaml
├── src
│  └── model
│     └── test.yaml
└── templates
   └── deployment.yaml
```

**Comment by luisdavim** [comment]
I guess I can move the other checks under the check for the path being under templates, I'll do some testing later today and open a pr.

**Comment by luisdavim** [comment]
I've opened https://github.com/towolf/vim-helm/pull/26 to address this issue, can you have a look?

**Comment by towolf** [comment]
Please retry, I merged #26.
